Modules/Segmentation/Voronoi/include/itkVoronoiSegmentationRGBImageFilter.hxx

```diff
@@ -177,32 +177,22 @@ VoronoiSegmentationRGBImageFilter<TInputImage, TOutputImage>::TestHomogeneity(In
     }
   }
 
-  bool ok = true;
   j = 0;
-  double savem, savev;
-  while (ok && (j < 3))
+  while ((j < 3))
```

Comment:
Thanks for making this a separate PR 👍  The extra pair of parentheses in `while ((j < 3))` still looks a bit funny to me, but no problem. As I wrote before: https://github.com/InsightSoftwareConsortium/ITK/pull/4977#discussion_r1852876529